src/pipx/animate.py

```diff
@@ -7,7 +7,7 @@
 from pipx.constants import WINDOWS
 from pipx.emojis import EMOJI_SUPPORT
 
-stderr_is_tty = sys.stderr.isatty()
+stderr_is_tty = sys.stderr.isatty() if sys.stderr else False
```

Comment:
```suggestion
stderr_is_tty = bool(sys.stderr and sys.stderr.isatty())
```

Same for other calls below.